.editorconfig

```diff
@@ -0,0 +1,22 @@
+# See http://editorconfig.org to read about the EditorConfig format.
+# - Automatically supported by VS2017+ and most common IDE or text editors.
+# - For older VS2010 to VS2015, install https://marketplace.visualstudio.com/items?itemName=EditorConfigTeam.EditorConfig
+
+# top-most EditorConfig file
+root = true
+
+# Default settings:
+# Use 4 spaces as indentation
+[*]
+indent_style = space
+indent_size = 4
+insert_final_newline = true
+trim_trailing_whitespace = true
+
+[src/overlay*{cpp,h}]
+indent_size = 3
+trim_trailing_whitespace = true
```

Comment:
Also this whole section is kinda unnecessary since the behavior is already defined in `[*]`.